dynamic_instrumentation/src/main.rs

```diff
@@ -312,6 +323,17 @@ fn cargo_wrapper(rustc_wrapper: &Path) -> anyhow::Result<()> {
         cmd.args(&["clean", "--package", root_package.name.as_str()]);
     })?;
 
+    if set_runtime {
+        cargo.run(|cmd| {
```

Comment:
What happens when the crate is already added? Does this exit with a non-zero status?

Reply:
No, it just overwrites it or does nothing, which is why I thought `--set-runtime` was clearer.